requirements.txt

```diff
@@ -1,3 +1,4 @@
-gunicorn
-requests
-pylint
\ No newline at end of file
+gunicorn==19.7.1
+requests==2.14.2
+pylint==1.8.4
+pandas==0.21
```

Comment:
can we put this into `conda-requirements` and move to a newer version?

Reply:
I would prefer all package were on the newest version possible.  We have very little code so we should start at the latest version.  As we accumulate code, upgrading will be more and more painful, so we want to start from a good place.

Can you describe more about what you mean when you say `conda` only has access to older versions of pandas? This page gives all packages in the linux-64 channel https://repo.continuum.io/pkgs/main/linux-64/ and it seems like they have versions that would work.

Reply:
if I run:
`conda install --file conda-requirements.txt --yes`

I get:
```
PackageNotFoundError: Package not found: '' Package missing in current linux-64 channels: 
  - pandas ==0.21
```

This might only be a problem for our versions of conda+requests, but we should all have the same versions and I want to avoid having to update all of them.

here's my `conda info`

```

               platform : linux-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : not installed
         python version : 2.7.13.final.0
       requests version : 2.14.2
       root environment : /opt/conda  (writable)
    default environment : /opt/conda/envs/test-env
       envs directories : /opt/conda/envs
          package cache : /opt/conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : None
           offline mode : False
```

Reply:
try 0.22.0

Reply:
My assumption here is that everyone else's setup in our group is the same, which might be in error (since I caused problems with mine). My conda isn't pointing to `.../pkgs/main/linux-64` so if everyone else's is, I'm happy to just add that channel and move on.

Reply:
Excellent point!  Let me check

Reply:
yes you are correct, everyone has the same environment.  I think because of that your solution is optimal, go ahead and merge!